src/main/resources/templates/index.html

```diff
@@ -1,61 +1,61 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
```

Comment:
It's hard to tell what changed here, and how it reduces technical debt.  The reviewer should add comments here, in Files Changed, along with citations of best practices used.

Reply:
Sorry my review was done but it seems I hadn't submitted it, just did it.